src/object.d

```diff
@@ -3257,8 +3252,6 @@ private size_t getArrayHash(in TypeInfo element, in void* ptr, in size_t count)
 }
 
 
-// @@@BUG5835@@@ tests:
-
```

Comment:
https://issues.dlang.org/show_bug.cgi?id=10574 is still open, https://issues.dlang.org/show_bug.cgi?id=5835 is closed.

> the tag should last forever as a reminder of why the test is there.

Only if there's something special about it.